README.md

```diff
@@ -38,14 +38,21 @@ if let tcxData = tcxData {
 
 ### Encode
 
-```swift
+Each Trackpoint time has to be unique. A per second increment from the previous. Below example uses workoutStartDateTime as placeholder. Change as appropriate on your codebase.
```

Comment:
This is better off by the `let track` declaration

Reply:
I'm not entirely sure what is meant here. Based on what I understand, there can be many tracks in a TCX file. Each Track, can then have multiple trackpoints inside)

eg:
track1 --> trackpoint1
            --> trackpoint2
...
            --> trackpointX
track2 --> trackpoint1
            --> trackpoint2
...
            --> trackpointX